lms/static/coffee/fixtures/calculator.html

```diff
@@ -1,13 +1,13 @@
 <ul>
   <li class="calc-main">
-    <a href="#" class="calc" title="Open Calculator" role="button" aria-controls="calculator_wrapper" aria-expanded="false">Open Calculator</a>
+    <button class="calc" title="${_('Open Calculator')" aria-controls="calculator_wrapper" aria-expanded="false">${_('Open Calculator')</button>
```

Comment:
It doesn't. I think the text is off-screen, which is probably why I added the title.

Going to hold off here until I get a yay or nay for removing this altogether...